### DISCUSSION TITLE
Jingle Playlist once every x minutes, only on scheduled times?

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#6687] [Q&A]
For the "end of daylight saving time" reminder in Germany on 2023-10-29, I try to set up a reminder. So I have a playlist "Zeitumstellung" with just one jingle and set it like this:

![Auswahl_286](https://github.com/AzuraCast/AzuraCast/assets/3706922/ca9f05a7-e088-4fa1-8578-6bd788d59a07)

![Auswahl_287](https://github.com/AzuraCast/AzuraCast/assets/3706922/d32c4fcc-c6aa-4582-b4fb-7ebe74d34b99)

Naively, I expected the jingle to **play every 5 minutes, but only during the scheduled times.**

But it didn’t, yet, and I’m sitting here at least an hour.

Questions:

1. **Is this supported?** (I _hope_ yes, it’s a very useful feature to play jingles only during certain times!)
2. **Must I do anything else**, like restart broadcasting for this to work?
3. Or do I have to **wait xx minutes until some periodic task hits in** and makes it working?

**Comment by Moonbase59** [comment]
Just crossed midnight, and now it played twice already. Coincidence?

**Comment by LordHelmchen666** [comment]
Normally it should work. I would "Reload Config" to be sure it is written to LS config and then look thru LS config what has been written just to be sure.

**Reply by Moonbase59** [reply]
Sounds good, cheers. I’ll try.

**Comment by Piratenkanon** [comment]
at changing something in Playlists you have to restart only the AutoDJ

**Comment by RadioHerzblut** [comment]
I had this phenomenon the other day with a playlist for the night.

The playlist should play from 00:00 AM - 05:00 AM .  There were already tracks in the queue before 00:00 AM.
I reset it a few times and now it works. I just don't know why !
